### DISCUSSION TITLE
Custom approval script

### DISCUSSION THREAD

**Discussion by hialvaro** [windmill-labs/windmill#1620] [Questions] [answered]
I am interested in creating a custom approval script that will open a new Jira issue. I don't see how I can get the approval URL from the approval script for the flow. Is there any way I can get the URL?

**Comment by rubenfiszel** [comment, marked as answer]
Yes, you can use `getResumeEndpoints` like in this example: https://hub.windmill.dev/scripts/windmill/1391/return-resume-and-cancel-endpoints-windmill

```typescript
import * as wmill from "https://deno.land/x/windmill@v1.85.0/mod.ts"

export async function main(approver?: string) {
  const endpoints = await wmill.getResumeEndpoints(approver)
  console.log(`Approval page at ${endpoints.approvalPage}`)
  return endpoints
}
```

**Reply by hialvaro** [reply]
Awesome, thanks
